CMakeLists.txt

```diff
@@ -116,6 +116,7 @@ set( TRACCC_THRUST_OPTIONS "" CACHE STRING
    "Extra options for configuring how Thrust should be used" )
 mark_as_advanced( TRACCC_THRUST_OPTIONS )
 thrust_create_target( traccc::Thrust ${TRACCC_THRUST_OPTIONS} )
+set_target_properties( traccc::Thrust PROPERTIES SYSTEM TRUE )
```

Comment:
This is generally good, but it only works with CMake 3.25+.

https://cmake.org/cmake/help/latest/prop_tgt/SYSTEM.html

What would be compatible with previous versions as well would be something like:

```cmake
get_target_property( _thurstIncludeDirs traccc::Thrust INTERFACE_INCLUDE_DIRECTORIES )
target_include_directories( traccc::Thrust SYSTEM INTERFACE ${_thurstIncludeDirs} )
```

Didn't actually test it though... :thinking:

Reply:
Oops, my mistake for thinking that something simple might actually be simple to do in CMake.

![image](https://user-images.githubusercontent.com/1165820/226664980-23299b74-8d47-44b5-b2d3-b4f25fa1a299.png)

Reply:
How about I set the `IMPORTED` property on it?

Reply:
😕 Did you remove your comment?

No, `IMPORTED` is not a property that could be set by hand.

https://cmake.org/cmake/help/latest/prop_tgt/IMPORTED.html

And I expect that the `traccc::Thrust` target would have it set to `TRUE` anyway.